### PR TITLE
Test undo only for new migrations 

### PR DESCRIPTION
**Pull request by smadbe**
- only try to undo the migrations which are new to this branch (not perfect but does the job)
- adapt new timeout to the new db dump

### CODE REVIEW

**Comment by zenovich** [comment]
According on the previous experience, most migration bugs are found by undoing old migrations, so I'd prefer trying to undo all the migrations if at least one new migration is added.

**Comment by smadbe** [comment]
It takes hours to run all undo migrations... and actually the only goal of testing the undo-migration is to check that in practice, if we deploy the new migration on prod, we can still just undo it... the goal is never to rollback 50 migrations.

**Comment by smadbe** [comment]
The main cases where it would not really work (and why there is "not perfect but does the job") is when:
- a migration is added not at the last one of history or a previous migration is modified.. but that should not happen
- when the undo of 2 migrations are conflicting when merged together on master (and branch were not merged/rebased before merging)

**Comment by zenovich** [comment]
> The main cases where it would not really work (and why there is "not perfect but does the job") is when:
> 
> * a migration is added not at the last one of history or a previous migration is modified.. but that should not happen
> * when the undo of 2 migrations are conflicting when merged together on master (and branch were not merged/rebased before merging)

The undo may be applied successfully without even doing anything. The only way to check it really changes the DB back is to undo all the previous migrations and run all the migrations again. There were cases where we got an error somewhere in very old migrations because the new one was not undone properly.

File: .circleci/config.yml [hunk 1]
```diff
@@ -120,7 +120,7 @@ jobs:
           no_output_timeout: 10m
       - run:
           name: Undo all migrations
-          command: for m in `git diff --name-only HEAD..origin/master | grep "db/migrations/"`; do if ! make db-migrate-undo; then break; fi; done
+          command: for m in db/migrations/*.sql; do if ! make db-migrate-undo; then exit 1; fi; done
```
Comment (smadbe) [inline .circleci/config.yml:123]: Fix one thing at a time.  Here you both fix a real bug, and revert my change. Useless to work in this PR is the commit revert the change.

File: conf/config.example2.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+database:
+  dbname: algorea_example
+
+domains:
+  -
+    domains: [127.0.0.1]
+    allUsersGroup: 3
+    tempUsersGroup: 2
```
Comment (smadbe) [inline conf/config.example2.yaml:8]: :-/

Comment (zenovich) [reply conf/config.example2.yaml]: fixed

**Review by zenovich — APPROVED** [review]
